autovalet_gazebo/launch/setup_slam_test.launch

```diff
@@ -4,14 +4,16 @@
 	<arg name="world_file" default="slam_val_dumpster.world"/>
 	<arg name="x" default="-18"/>
 	<arg name="y" default="-9"/>
+	<arg name="theta" default= "0"/>
 
   	<include file="$(find autovalet_gazebo)/launch/parking_lot.launch">
   		<arg name="joystick" value="false"/> 
   		<arg name="world_file" value="$(arg world_file)"/>
 		<arg name="x" value="$(arg x)"/>
 		<arg name="y" value="$(arg y)"/>
+		<arg name="yaw" value= "$(arg theta)"/>
```

Comment:
why do we need this?

Reply:
this file is old. why are we using this now?

Reply:
this is for intermediate testing.. we are using this everywhere even now actually. Get's called around in the state machine simulation too! It helps spawn the robot in any orientation that we want